package.json

```diff
@@ -3,19 +3,21 @@
   "version": "0.5.0",
   "license": "MIT",
   "scripts": {
-    "build": "yarn generate:definitions && tsc",
+    "prebuild": "npm-run-all --parallel generate:*",
```

Comment:
will need rebase after https://github.com/pagopa/io-functions-admin/pull/63 is merged